database/migrations/default/1750435376415_ems_person_match_auto/up.sql

```diff
@@ -0,0 +1,437 @@
+--
+-- add person_match_status column
+--
+ALTER TABLE ems__incidents
+ADD COLUMN person_match_status text DEFAULT 'unmatched' CHECK (
+    person_match_status IN (
+        'unmatched',
+        'unmatched_by_manual_qa',
+        'matched_by_automation',
+        'matched_by_manual_qa',
+        'multiple_matches_by_automation'
+    )
+),
+ADD COLUMN matched_person_ids integer [],
+ADD COLUMN _match_event_name text
+-- add constraint to ensure event name is always reset to null
+CHECK (_match_event_name IS NULL),
+-- add constraint to ensure that crash_pk is never null if person_id is not null
+ADD CONSTRAINT check_person_id_crash_pk_dependency CHECK (
+    person_id IS NULL
+    OR crash_pk IS NOT NULL
+);
+
+COMMENT ON COLUMN ems__incidents.person_match_status IS 'The status of the CR3 person record match';
+
+COMMENT ON COLUMN ems__incidents.matched_person_ids IS 'The IDs of the CR3 person records that were found to match this record';
```

Comment:
so multiple `person_ids` can be found to match one EMS patient record? it would be good to add testing for this in the PR. 

also similar to the comment for `matched_crash_pks` _("The IDs of crashes that were found to match this record. Set via trigger, always kept up to date regardless of EMS crash match status.")_  you could state here that this is set via trigger and always kept up to date regardless of person match status.